changelog/unreleased/bug-fixes/1947--query-worker-deadlock.md

```diff
@@ -0,0 +1,3 @@
+Queries that resulted in zero results sometimes caused query workers to become
+stuck, possibly resulting in a deadlock when this affected all query workers.
+This now works as expected again.
```

Comment:
Since this problem was introduced after the last release I would skip this message and only add the PR number to the entry for #1929.

Reply:
Makes sense.